src/webgui/src/webgui/image_pub.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import sys
+import rospy
+import cv2
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+import rospy
+import time
+
+
+class image_pub:
+	""" Class for publishing images from video source
+
+    Attributes:
+        pub (rospy.Publisher): Publisher that publishes images to topic.
+        bridge (CvBridge): CvBridge instance that converts cv2 images to imgmsg
+        cam (cv2.VideoCapture): Gets frames from video source
+	"""
+	def __init__(self, image_topic, video_source):
+		"""Init method for image_pub
+		
+        Args:
+            image_topic (str): Topic to publish to.
+            video_source (str): Path to video file
+
+        """
+		self.pub = rospy.Publisher(image_topic, Image, queue_size=10)
+		self.bridge = CvBridge()
+		self.cam = cv2.VideoCapture(video_source)
+	
+	def publish_images(self):
+		"""Retrieves and publishes images from video source"""
+		
+		while self.cam.isOpened() and not rospy.is_shutdown():
+			meta, frame = self.cam.read()
+			msg_frame = self.bridge.cv2_to_imgmsg(frame)		
+			self.pub.publish(msg_frame)
+
+def main(args):
+	# get params from rosparam
+	node_name = rospy.get_param('imgpub/node_name', 'imgpub_node')
```

Comment:
Node name can automatically be changed on the command line, so maybe we don't need it in rosparam. 
See "Special Keys" - http://wiki.ros.org/Remapping%20Arguments

Reply:
So should I just replace it with `node_name = __name`?